### ISSUE TITLE
[new hparams] optimzer, learning rate etc. hparams support ?

### ISSUE THREAD

**Issue by lsq357** [BrikerMan/Kashgari#159]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by BrikerMan** [comment]
1. [custom Hyper-parameters support](https://kashgari.bmio.net/tutorial/text-classification/#adjust-models-hyper-parameters)
2. You can build a model, then re-compile with new optimizer with any lr you needs. Here is an example:

```
from kashgari.tasks.classification import BiLSTM_Model

model = BiLSTM_Model()
model.build_model(train_x, train_y)
model.compile(optimizer=Adam(lr=0.0001))
```

Here is the [API documents](https://kashgari.bmio.net/api/tasks.classification/#compile_model).
